pkg/features/kube_features.go

```diff
@@ -1250,7 +1250,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	genericfeatures.APIServingWithRoutine: {Default: true, PreRelease: featuregate.Beta},
 
-	genericfeatures.ConsistentListFromCache: {Default: false, PreRelease: featuregate.Alpha},
+	genericfeatures.ConsistentListFromCache: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
what is the behavior of this when we aren't running against the minimum etcd version we expect?

Reply:
Ups, this is still a draft waiting for https://github.com/kubernetes/kubernetes/pull/123264 to be merged and ensure minimal etcd version.